test/check-application

```diff
@@ -2051,6 +2052,31 @@ class TestFiles(testlib.MachineCase):
         def dir_file_count(directory: str) -> int:
             return int(m.execute(f"find {directory} -mindepth 1 -maxdepth 1 | wc -l").strip())
 
+        def assert_upload_alert(files: List[str], owner_group: Optional[str] = None, *,
+                                close: bool = True, pixel_test: bool = False) -> None:
+            title = ""
+            description = ""
+            if len(files) > 1:
+                title = "Files uploaded"
+                description = f"{len(files)} files"
+            else:
+                title = "File uploaded"
+                description = files[0]
+
+            if owner_group:
+                # permission = self.stat("%A", f"/home/admin/{os.path.basename(files[0])).lstrip('-')
+                description += f"Uploaded as {owner_group}, rw- r-- r--"
```

Comment:
hardcoding the permissions is acceptable for a test, at least until we get an image with a default 077 umask or so. But either use the commented out permission or drop that commented code.

Reply:
Right, I wanted to make it dynamic but then found out I needed to nicely space it and didn't go through with it.